Symfony/CS/Fixer/PSR2/BracesFixer.php

```diff
@@ -194,7 +196,13 @@ function ($item) {
                         // and it is not a `${"a"}->...` and `${"b{$foo}"}->...` situation
                         !($nestToken->equals('}') && $tokens[$nestIndex - 1]->equalsAny(array('"', "'", array(T_CONSTANT_ENCAPSED_STRING))))
                     ) {
-                        if ($nextNonWhitespaceNestToken->isGivenKind($this->getControlContinuationTokens())) {
+                        if (
+                            $nextNonWhitespaceNestToken->isGivenKind($this->getControlContinuationTokens()) ||
+                            (
+                                $nextNonWhitespaceNestToken->isGivenKind(T_WHILE) &&
+                                $tokensAnalyzer->isWhilePartOfDoWhile($nextNonWhitespaceNestIndex)
```

Comment:
This fixes bug with setting token as changed when it not need to be changed (on 1.x line we add more spaces into T_WHITESPACE and remove it later, now we don't do that).
It will change nothing on 1.x line, and is needed on 2.x line only if we choose a way of marking tokens as changed.